lib/server.js

```diff
@@ -177,14 +175,67 @@ server.route({
   }
 });
 
+server.route({
+  path: '/{serverpath*}',
+  method: 'DELETE',
+  config: {
+    validate: {
+      headers: Joi.object().keys({
+        cookie: Joi.string().regex(/^AuthSession=[A-Za-z0-9\-_]+/).required()
+      }).options({allowUnknown: true})
+    }
+  },
+  handler: function(req, reply) {
+    request({
+      uri: config.couchDbSessionUrl,
+      json: true,
+      headers: {
+        cookie: req.headers.cookie
+      }
+    }, function(err, response, body) {
+      if (err || (response.statusCode !== 400 && response.statusCode !== 200)) {
+        reply(Boom.badImplementation('database connection error'));
+        return;
+      }
+
+      if (response.statusCode === 400) {
+        reply(Boom.badRequest(body.reason));
+        return;
+      }
+
+      if (body.userCtx.name === null) {
+        reply({ok: true});
```

Comment:
While adding a respective test case, I figured that it would be better to return with status code `401` (unauthorized) instead, as we don't (because we can't) clear any other potential still valid session. What do you think?